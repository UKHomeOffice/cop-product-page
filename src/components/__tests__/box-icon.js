import React from "react";
import renderer from "react-test-renderer";
import BoxIcon from "../box-icon";

describe("Box-icon", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<BoxIcon colour={"red"}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("Returns the correct box icon 'colour' prop passed in", () => {
    const tree = renderer
      .create(<BoxIcon colour={"red"}/>);
    const testInstance = tree.root;
    expect(testInstance.props).toHaveProperty('colour', 'red');
  });
});