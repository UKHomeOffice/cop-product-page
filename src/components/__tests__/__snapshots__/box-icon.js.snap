// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box-icon renders correctly 1`] = `
<i
  className="icon-box"
  style={
    Object {
      "backgroundColor": "red",
    }
  }
/>
`;
